### PR TITLE
Migration to V5

### PR DESCRIPTION
**Pull request by ma-shulgin**
Migrated and tested

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,25 +1,40 @@
...
+WORKDIR /squid
+COPY --from=deps /squid/package.json .
+COPY --from=deps /squid/package-lock.json .
+COPY --from=deps /squid/node_modules node_modules
+COPY --from=builder /squid/lib lib
 ADD db db
-ADD manifest.yml .
 ADD schema.graphql .
-ADD .env .
+# TODO: use shorter PROMETHEUS_PORT
```
Comment (vikiival) [inline Dockerfile:29]: @ma-shulgin still todo?

File: Dockerfile [hunk 2]
```diff
@@ -1,25 +1,40 @@
...
+ENV PROCESSOR_PROMETHEUS_PORT 3000
+EXPOSE 3000
+EXPOSE 4000
```
Comment (vikiival) [inline Dockerfile:32]: You use `4350` in `.env`. Isn't that an issue?

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,52 @@
+process: build
```
Comment (vikiival) [inline Makefile:1]: Kjut. I recommend `casey/just`

File: manifest.yml [hunk 4]
```diff
@@ -1,25 +0,0 @@
-version: '4.0'
```
Comment (vikiival) [inline manifest.yml:1]: this does not seem legit or?

File: src/mappings/index.ts [hunk 5]
```diff
@@ -32,16 +32,17 @@ import {
...
 import { create, get } from './utils/entity'
 import { fetchMetadata } from './utils/metadata'
-import { DatabaseManager } from '@subsquid/hydra-common'
+import { Store} from '@subsquid/substrate-processor'
 import {updateCache} from './utils/cache'
 
+
 export async function handleRemark(context: Context): Promise<void> {
-  const remark = new System.RemarkCall(context.extrinsic).remark
+  const remark = new SystemRemarkCall(context).asV1020.remark
```
Comment (vikiival) [inline src/mappings/index.ts:45]: Wouldn't be this issue for older blocks?

**Review by vikiival — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (ma-shulgin) [reply Dockerfile:29]: I think could be deleted. Just copied from [template](https://github.com/subsquid/squid-template)

File: Dockerfile (hunk 2 above)
Comment (ma-shulgin) [reply Dockerfile:32]: I think it's better not to change it. Also copied from [template](https://github.com/subsquid/squid-template). Deployment works well with it

File: Makefile (hunk 3 above)
Comment (ma-shulgin) [reply Makefile:1]: Thanks! Will check this out

File: src/mappings/index.ts (hunk 5 above)
Comment (ma-shulgin) [reply src/mappings/index.ts:45]: Nah, that's only one version of remark call spec in Runtime. Btw, new squid supports type safe types, you can check this [here](https://github.com/subsquid/squid-template/blob/main/src/processor.ts#L63)

Comment (vikiival) [reply src/mappings/index.ts:45]: Perfect!

**Pull request merged**